src/tests/unit/tests/DetailsView/components/issues-table.test.tsx

```diff
@@ -164,6 +166,9 @@ describe('IssuesTableTest', () => {
                 issueDetailsTextGenerator: null,
                 windowUtils: null,
                 bugActionMessageCreator: null,
+                environmentInfoProvider: null,
```

Comment:
I guess we need this because of type changes. None of this props show up on the snapshots though. It would be interesting to check if we need to make sure they show up and if we don't need that maybe find a way to not add this (as in, avoid adding new props -because type changes- that don't show up on the snapshots, making coder live easier in the future)

Reply:
I think this is a good point. I think we should not have to touch this file when we change the deps on a child component. I think it is good to cast deps to 'IssuesTableDeps' and leave the deps that is not directly used in this component.